extension/package-lock.json

```diff
@@ -1,8 +1,2663 @@
 {
   "name": "minerva",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
It appears that you are using a different version of npm which is compatible with the existing version in repo but exhibits a different runtime behavior. I am not able to trace down the root cause and verified that the version i am using right now is the latest. For now, let's set this file aside by revert the changes in this file back to existing content. Your installed modules should not be affected by this